cirq/study/result.py

```diff
@@ -150,6 +150,8 @@ def measurements(self) -> Dict[str, np.ndarray]:
 
     @property
     def repetitions(self) -> int:
+        if not self.measurements or not self.measurements.values():
```

Comment:
I think the empty dict is Falsey so we could just use this: 
```suggestion
        if not self.measurements:
```

Reply:
Ah. Thanks.